specification/common-types/resource-management/v5/networksecurityperimeter.json

```diff
@@ -121,7 +149,7 @@
           "x-ms-arm-id-details": {
             "allowedResources": [
               {
-                "resourceType": "Microsoft.Network/networkSecurityPerimeters"
+                "type": "Microsoft.Network/networkSecurityPerimeters"
```

Comment:
Is this supposed to be `type` or `allowedResources`? The documentation here seems to contradict itself:

https://azure.github.io/autorest/extensions/#x-ms-arm-id-details

Reply:
what I see in the documentation is that it needs to be allowedResources and the schema for allowedResources is type so the structure should be like this example
```json
"MyExampleType": {
  "properties": {
    "vnetId": {
      "type": "string",
      "format": "arm-id",
      "x-ms-arm-id-details": {
        "allowedResources": [
          {
            "type": "Microsoft.Network/virtualNetworks"
          }
        ]
      }
    }
  }
}
```

Reply:
Yeah you're right. The doc just has some formatting issues where its hard to follow and examples that seem a little confusing because they seem to lump type and scope together. Sorry!